doc/docs/modules/ROOT/pages/producer-configuration.adoc

```diff
@@ -45,6 +46,13 @@ You can tune the custom property `kafka.topic.discovery.polling.interval` in ord
 periodically check for new topics into the Kafka cluster so the plugin will be able
 to send events to the defined topics.
 
+
+With `kafka.streams.log.compaction.strategy=delete` will be generated a sequence of unique keys with Neo4j Streams Source.
+instead with `kafka.streams.log.compaction.strategy=compact` the keys will be adapted to enable
+https://kafka.apache.org/documentation.html#compaction[Log Compaction] on the Kafka side.
+Please note that delete strategy does not actually delete records, it has this name  to match the topic config `cleanup.policy=delete/compact`.
+Namely, the operations which will involve the same nodes or relationships, will have the same key.
+
```

Comment:
Please add the docs regarding the key structure for nodes and relationships for both use-case with/without unique contraints